__tests__/cmds/versions.test.js

```diff
@@ -103,31 +105,27 @@ describe('rdme versions*', () => {
     });
 
     it('should create a specific version', async () => {
-      promptHandler.createVersionPrompt.mockResolvedValue({
-        is_stable: true,
-        is_beta: false,
-        from: '1.0.0',
-      });
-
+      const args = { key, version, fork: '1.0.0', main: false, beta: true, public: true };
       const mockRequest = getApiNock()
-        .get('/api/v1/version')
-        .basicAuth({ user: key })
-        .reply(200, [{ version }, { version }])
-        .post('/api/v1/version')
+        .post('/api/v1/version', {
```

Comment:
We weren't fully testing out the expected API payload. Wish there were a way for us to configure Nock to require this...